contrib_bots/bots/foursquare/foursquare.py

```diff
@@ -9,13 +9,13 @@
 from six.moves import range
 
 home = expanduser('~')
-CONFIG_PATH = home + '/zulip/contrib_bots/bots/foursquare/FourSquareBot/settings.ini'
+CONFIG_PATH = home + '/zulip/contrib_bots/bots/foursquare/foursquare.config'
 
 def get_api_key():
-    # settings.ini must have been moved from
-    # ~/zulip/contrib_bots/bots/foursquare/FourSquareBot/settings.ini into
-    # ~/settings.ini for program to work
-    # see doc.md for more information
+    # foursquare.config must have been moved from
```

Comment:
@tommyip Yeah, I understand that. I was simply following the guidelines as specified in the issue #3427. If you suggest .cfg/.ini, I can modify accordingly.